static/js/app.js

```diff
@@ -368,7 +368,7 @@ function showBannersIfNecessary() {
     return;
   }
 
-  const bg = dashboardBanners.querySelector(".banner-gradient-bg");
+  const bg = dashboardBanners.querySelector(".banner-gradient-bg.addon");
```

Comment:
Bug fix to hide ONLY the add-on banner if a user has it installed. Previously, this would also hide the promo to upgrade.